eclair-core/src/main/scala/fr/acinq/eclair/Features.scala

```diff
@@ -10,6 +10,9 @@ import fr.acinq.bitcoin.BinaryData
   * Created by PM on 13/02/2017.
   */
 object Features {
+  val OPTION_DATA_LOSS_PROTECT_MANDATORY = 0
```

Comment:
We must also make sure that `areSupported()` will return true if bit OPTION_DATA_LOSS_PROTECT_MANDATORY is set.